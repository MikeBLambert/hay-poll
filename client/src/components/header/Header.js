import React from 'react';
import { rootLinks } from '../../routes/routes';
import styles from './Header.css';

export default function Header() {
  return (
    <div className={styles.header}>
      <nav className={styles.nav}>{rootLinks()}</nav>
    </div>
  );
}
