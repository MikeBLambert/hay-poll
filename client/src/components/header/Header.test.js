import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header component', () => {
  it('renders a header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  })
})