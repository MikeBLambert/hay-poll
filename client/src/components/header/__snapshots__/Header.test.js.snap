// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders a header component 1`] = `
<div
  className="header"
>
  <nav
    className="nav"
  >
    <Link
      key="0"
      replace={false}
      to="/"
    >
      home
    </Link>
  </nav>
</div>
`;
